sycl/include/sycl/ext/intel/experimental/esimd/memory.hpp

```diff
@@ -718,8 +718,10 @@ template <typename T, int NElts = 1,
           lsc_data_size DS = lsc_data_size::default_size,
           cache_hint L1H = cache_hint::none, cache_hint L3H = cache_hint::none,
           int N, typename AccessorTy>
-__ESIMD_API std::enable_if_t<!std::is_pointer_v<AccessorTy>,
-                             __ESIMD_NS::simd<T, N * NElts>>
+__ESIMD_API std::enable_if_t<
+    !std::is_pointer_v<AccessorTy> &&
```

Comment:
Is it required to explicitly check "not local accessor" in this cases (and similar others)? Would it not match without it?
(Sorry if this was already discussed somewhere else).